tests/gnmi/conftest.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import shutil
+
+from tests.common.helpers.assertions import pytest_require as pyrequire
+from tests.common.helpers.dut_utils import check_container_state
+from helper import GNMI_CONTAINER_NAME, apply_cert_config, create_ext_conf
+from tests.generic_config_updater.gu_utils import create_checkpoint, rollback
+
+SETUP_ENV_CP = "test_setup_checkpoint"
+
+
+@pytest.fixture(scope="function", autouse=True)
+def skip_non_x86_platform(duthosts, rand_one_dut_hostname):
+    """
+    Skip the current test if DUT is not x86_64 platform.
```

Comment:
>x86_64 [](http://example.com/codeflow?start=40&length=6)

Is it possible to test other platforms?

Reply:
https://github.com/sonic-net/sonic-mgmt/pull/7081
Zain's PR only supports x86 platform.
It's possible to test other platform, and we need to install GNMI client in sonic-mgmt container.